skops/card/_model_card.py

```diff
@@ -129,6 +134,14 @@ class Card:
     >>> model_card = card.Card(model)
     >>> model_card.metadata.license = "mit"
     >>> y_pred = model.predict(X)
+    >>> model_card.add_metrics(**{"accuracy": # doctest: +ELLIPSIS
+    ... accuracy_score(y, y_pred), # doctest: +ELLIPSIS
+    ... "f1 score": f1_score(y, y_pred, average='micro')}) # doctest: +ELLIPSIS
```

Comment:
IMO, this looks a bit strange, how about the formatting below?

```suggestion
    >>> model_card.add_metrics(**{
    ...     "accuracy": accuracy_score(y, y_pred),
    ...     "f1 score": f1_score(y, y_pred, average="micro"),
    ... })
```

Also, not sure if `# doctest: +ELLIPSIS` is required. I removed it and it still works for me.